pennylane/math/multi_dispatch.py

```diff
@@ -507,6 +507,43 @@ def stack(values, axis=0, like=None):
     return np.stack(values, axis=axis, like=like)
 
 
+def einsum(indices, *operands, like=None):

```

Comment:
I've added this function in another PR that just got merged: https://github.com/PennyLaneAI/pennylane/pull/2615, so you can probably remove this.

We can still keep your changes to this function in [QIM-2] though.